addon/components/frost-radio-button.js

```diff
@@ -51,32 +58,27 @@ export default Component.extend(PropTypeMixin, {
 
   // == Computed properties  ===================================================
 
-  groupId: readOnly('parentView.id'),
-  groupValue: readOnly('parentView.value'),
-  onChange: readOnly('parentView.onChange'),
-
-  @computed('groupValue', 'value')
+  @computed('selectedValue', 'value')
   /**
    * Determine checked state
-   * @param {String} groupValue - which radio button in the group is set
-   * @param {String} value - is this radio button selected
-   * @returns {Boolean} whether this radio button is checked
+   * @param {String} selectedValue - which radio button in the group is selected
+   * @param {String} value - radio button value
+   * @returns {Boolean} whether this radio button is checked or not
    */
-  checked (groupValue, value) {
-    return groupValue === value
+  checked (selectedValue, value) {
+    return selectedValue === value
   },
 
-  @computed('value')
+  @computed('value', 'receivedHook')
   /**
    * Determine hook name for radio-button
    * @param {String} value - radio button's value
+   * @param {String} receivedHook - hook received from parent
    * @returns {String} the concatenated hook name
    */
-  hook (value) {
-    const radioGroupHook = get(this, 'parentView.hook')
-    if (radioGroupHook) {
-      return `${radioGroupHook}-button-${value}`
-    }
+  hook (value, receivedHook) {
```

Comment:
could we also set a hookQualifier property [set to 'value'] here so we stay consistent with the qualifier usage across the addon?